common/default_chains_mainnet.go

```diff
@@ -20,7 +20,7 @@ func BscMainnetChain() Chain {
 func ZetaChain() Chain {
 	return Chain{
 		ChainName: ChainName_zeta_mainnet,
-		ChainId:   101,
+		ChainId:   7000,
```

Comment:
70000

Reply:
This is for mainnet , added 70000 for mock mainnet